sensor/extrinsic_calibration_manager/launch/aip_xx1/map_based.launch.xml

```diff
@@ -17,6 +17,7 @@
     <include file="$(find-pkg-share extrinsic_calibration_manager)/launch/$(var sensor_model)/map_based_sensor_kit.launch.xml">
       <arg name="vehicle_id" value="$(var vehicle_id)" />
       <arg name="use_calibration_area_map" value="$(var use_calibration_area_map)" />
+      <arg name="fitness_score_threshold" value="$(var fitness_score_threshold)" />
```

Comment:
Do you revert this change too?

Reply:
```suggestion
```

Reply:
very sorry. There was unfinished business.
I will correct this.